modules/oadp-setting-resource-limits-and-requests.adoc

```diff
@@ -37,4 +37,4 @@ spec:
             cpu: 500m
             memory: 256Mi
 ----
-<1> Specify the node selector to be supplied to Velero podSpec
+<1> Specify the node selector to be supplied to Velero podSpec.
```

Comment:
this will remove the extra (1), will it?
how can i preview :) ?

Reply:
@amastbau We will only know after OCP publishes it. We are trying to see if it works. Please authorize so that we can move it to OCP review and merge.

Reply:
approved then. @CarmiWisemon  sorry for the delay

Reply:
Thank you!

Reply:
@CarmiWisemon build previews are available in the comments of every pull request, you just need to navigate to the appropriate build (OCP, ROSA, etc) and page (virtualization, backup, installing OADP). Additionally, we require build previews as part of the review process - it is in the description upon creating the PR. 

Here is 4.12 docs:
https://docs.openshift.com/container-platform/4.12/virt/backup_restore/virt-installing-configuring-oadp.html#oadp-setting-resource-limits-and-requests_virt-installing-configuring-oadp

And here is the build preview for this pull request:
https://59458--docspreview.netlify.app/openshift-enterprise/latest/virt/backup_restore/virt-installing-configuring-oadp.html#oadp-setting-resource-limits-and-requests_virt-installing-configuring-oadp

I don't see an additional <1>. What problem are you trying to solve?

Reply:
@sheriff-rh I was looking at that as well, there is an extra `1` in the docs on the customer portal. https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html/backup_and_restore/application-backup-and-restore#configuring-dpa-aws

I know we are suppose to have periods at the end of sentences in the callouts. So it does seem like the missing period is causing the issue. WDYT?